setup.py

```diff
@@ -81,23 +81,17 @@
     platforms='any',
     classifiers=[
         "Programming Language :: Python",
+        'Programming Language :: Python :: 2',
         "Programming Language :: Python :: 2.7",
-        # 'Development Status :: 1 - Planning',
-        # 'Development Status :: 2 - Pre-Alpha',
-        'Development Status :: 3 - Alpha',
-        # 'Development Status :: 4 - Beta',
-        # 'Development Status :: 5 - Production/Stable',
-        # 'Development Status :: 6 - Mature',
-        # 'Development Status :: 7 - Inactive',
-        "Environment :: Other Environment",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        'Programming Language :: Python :: Implementation :: CPython',
+        'Programming Language :: Python :: Implementation :: Cython',
         "Intended Audience :: Developers",
-        "License :: OSI Approved :: Apache Software License",
```

Comment:
The license is still Apache (as it was in urlparse4, and still is for this repo), right? Any reason why you want to remove this?

Reply:
Oh I was just wondering which license we should put for this repo (in case you guys might want to change that)

Reply:
+1 to keep Apache, though github suggests that "Licensed works, modifications, and larger works may be distributed under different terms and without source code.", so changing it is likely not an issue.